swap/swap.go

```diff
@@ -130,7 +129,7 @@ func createOwner(prvkey *ecdsa.PrivateKey) *Owner {
 
 // DeploySuccess is for convenience log output
 func (s *Swap) DeploySuccess() string {
-	return fmt.Sprintf("contract: %s, owner: %s, deposit: %v, signer: %x", s.owner.Contract.Hex(), s.owner.address.Hex(), s.params.InitialDepositAmount, s.owner.publicKey)
+	return fmt.Sprintf("contract: %s, owner: %s, deposit: %v, signer: %x", s.GetParams().ContractAddress.Hex(), s.owner.address.Hex(), s.params.InitialDepositAmount, s.owner.publicKey)
```

Comment:
Not related to this PR, but your change reminds me of it: `signer: %x........,s.owner.publickey` actually prints garbage. Can you open an issue where we'd print the hex value of it? Thanks